index.html

```diff
@@ -0,0 +1,76 @@
+<!DOCTYPE html>
+<html>
+<head>
+<meta charset="utf-8">
+
+<title>Odin's html/css project</title>
+<link rel="stylesheet" href="style.css">
+
+</head> 
+<body>
+    <div class="body">
+        <div class="part1">
+           <div class="logo-web">
+            <div class="logo">some logo</div>
+            <div class="web1">
+                <div class="best-web">This is my awesome website</div>
+                <div class="subtext" >get in and feel it</div>
+                <button class="button1">sign up</button>
+                </div>
+            
+             </div>
+             <div class="link-img">
+                <div class="header-links">
+                    <a href="google.com" class="header-links"> link one </a>
+                    <a href="google.com" class="header-links">  link two </a>
+                    <a href="google.com" class="header-links">  link three </a>
+                </div> 
+                 <div class="web-img">
+                 <div class="website-image"><img src="flow.jpeg" text="this is a placeholder for an image"></div>
+                 </div>
+                </div>
+              </div>
+          <div class="part2">
+              <div class="text1">you are getting  back this</div>
+            <div class="images">
+                <div class="img1">
+                    <img src="aura.jpeg">
+                    <div class="text11">aura</div>
+                </div>
+                 <div class="img2">
+                     <img src="balance.jpeg">
+                     <div class="text2">balance</div>
+                 </div>
+                 <div class="img3">
+                     <img src="clarity.jpeg">
+                     <div class="text3">clarity</div>
+                 </div>
+                 <div class="img4">
+                     <img src="feel nature.jpeg">
+                     <div class="text4">nature</div>
```

Comment:
Kindly put alt(alternative text) for your image. Alternative text is a good way to show what you are trying to show in case the image took a long time to load.

Reply:
Perfect:ok_hand: